core/types/block.libevm.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2024 the libevm authors.
+// Copyright 2024-2025 the libevm authors.
```

Comment:
Trying to up my meme game in PR reviews

![9j0xwf](https://github.com/user-attachments/assets/973e8c17-b54d-40fa-a127-5f9cf6e291be)